src/browser/integrations/onerror.ts

```diff
@@ -28,8 +28,6 @@ export function onError(_window: any = window): Plugin {
             return
           }
 
-          if (!client.config.enableUncaught) { return }
-
           if (line === 0 && /Script error\.?/.test(msg)) {
             // See https://developer.mozilla.org/en/docs/Web/API/GlobalEventHandlers/onerror#Notes
             client.logger.info('Ignoring cross-domain script error: enable CORS to track these types of errors', arguments)
```

Comment:
I think we shouldn't log this message if `enableUncaught` is `false`, or we could change it to a _debug_ log.

Reply:
Good idea, I'm going to add an `enableUncaught` check here. I think users should be aware of these errors since they can improve their error monitoring by adding the CORS config.